daemon/graphdriver/quota/projectquota.go

```diff
@@ -194,6 +221,24 @@ func setProjectQuota(backingFsBlockDev string, projectID uint32, quota Quota) er
 	return nil
 }
 
+// getQuotaStat - get the quota stat
+// return 2 means quota is on
```

Comment:
I just take this https://kernel.googlesource.com/pub/scm/utils/quota/quota-tools/+/master/quotasys.c#1123  as a reference